client/src/pages/DonationFormReviewPage.tsx

```diff
@@ -75,6 +87,11 @@ const DonationFormReviewPage: FunctionComponent<Props> = (props: Props) => {
         Promise.all(createDonationFormPromises)
             .then((results) => {
                 const donationFormIds = results.map((result) => result.data.donationForm._id);
+                sendConfirmationEmail({
+                    variables: {
+                        ids: donationFormIds
+                    }
+                });
                 console.log(donationFormIds);
```

Comment:
nit: remove this console.log
I put it in because the linter was complaining about `donationFormIds` being unused 😅

Reply:
Removed.